docs/props.md

```diff
@@ -0,0 +1,23 @@
+# API Reference
+
+## Props
+
+| **Name** | **Type** | **Default** | **Description** | **Code Example** |
+|----------|----------|----------|----------|----------|
+| `current`                 | `string` or `Date`  |          | The current date that is displayed in the calendar.         | `<Calendar current={new Date()}/>`         |
```

Comment:
The type is 'string', in a specific format 'YYYY-MM-dd', Date is invalid

Reply:
Thinking we might want to remove this file now unless you like the API reference to be a separate section in the TOC of the docusaurus docs?

Reply:
No. I prefer to remove it so we won't have to maintain another file. We have the .api.json files for anyone that is looking at the source code and these are used to generate the docs

Reply:
Yes, makes sense to remove it - doesn't serve any purpose.